components/tree/private/render-branch.jsx

```diff
@@ -224,9 +224,9 @@ const getTabIndex = (props) => {
 
 const formatUrl = (props) => {
   const pathArr = props.node.id.split(/(items|item)(.+)/)
-  const itemPath = pathArr[2].replace(/\//g, '@')
+  const itemPath = pathArr[2] ? pathArr[2].replace(/\//g, '@') : ''
   const formattedPath = itemPath.replace('@limio@catalogs@1@tree', '@limio@catalogs@1@items')
```

Comment:
@benny-o  - do we still need to do this @limio@catalogs@1@tree ?  Can we look at why this is necssary